contracts/utils/TokenRegistry.sol

```diff
@@ -120,7 +120,7 @@ contract TokenRegistry is Ownable {
         tokenAddresses.push(_token);
```

Comment:
what is the purpose of the this contract?

Reply:
It's used by one script to get the token list of 0x. Not currently used, but is useful to keep